Utils/Dataflow/016_task2es/task2es.py

```diff
@@ -129,6 +129,59 @@ def get_category(row):
     return categories
 
 
+def input_events(data):
+    """ Calculate derived value 'input_events'.
+
+    For EVNT tasks:
+     * n_files_to_be_used  -> nFilesToBeUsed * (nEventsPerJob|1) / nFilesPerJob
+     * !n_files_to_be_used -> processed_events
```

Comment:
This formatting is somewhat puzzling. Is there some reason for it, for example, a documentation constructor or some other non-human interpreter demanding such format? Otherwise:

1. Why is name format different in the right part of line 136? Commit message and the code itself use `some_var_name` format, yet here it's `someVarName`.
2. What's `|1`? I know that "!x" = "not x" and with such interpretation line 137 correlates with the code and the commit message. However, if I presume "x|y" = "x or y (under some condition)", then it contradicts the code:

            try:
                events_per_job = int(data.get('n_events_per_job'))
            except TypeError:
                events_per_job = 0
 
If this is a typo and should be `|0`, then shouldn't `nFilesPerJob` be `(nFilesPerJob|1)`?

Reply:
@Evildoor , sorry for the late reply.

> Why is name format different in the right part of line 136? Commit message and the code itself use `some_var_name` format, yet here it's `someVarName`.

`some_var_name` is an exact name of document field in ES -- the value, that is derived from some other values. 
`someVarName` etc. -- I believe I just wanted to save some space. The line is long enough, and usage of underscores would make 10 extra symbols. But if it would look better I can change it.

> What's ` |1`? I know that "!x" = "not x" and with such interpretation line 137 correlates with the code and the commit message. However, if I presume "x|y" = "x or y (under some condition)", then it contradicts the code: <...>
> If this is a typo and should be `|0`, then shouldn't `nFilesPerJob` be `(nFilesPerJob|1)`?

Right, it should be read as "x or y". Hm, there must be a typo, as you said: should be `(nFilesPerJob|1)` indeed. Will check again and fix it.

`|0` part, seen in code, was added "just in case", as we can never be sure there is no NULL in Oracle data. If this field is NULL -- then it is taken for 0; I don\`t think it should be explicitly specified in docs, it is just a (more or less) standard way to convert NULL to int.

Reply:
> some_var_name is an exact name of document field in ES -- the value, that is derived from some other values.
> someVarName etc. -- I believe I just wanted to save some space. The line is long enough, and usage of underscores would make 10 extra symbols. But if it would look better I can change it.

I do believe that this change is necessary, because the current, non-unified version is rather puzzling.

> Right, it should be read as "x or y". Hm, there must be a typo, as you said: should be (nFilesPerJob|1) indeed. Will check again and fix it.
> |0 part, seen in code, was added "just in case", as we can never be sure there is no NULL in Oracle data. If this field is NULL -- then it is taken for 0; I don`t think it should be explicitly specified in docs, it is just a (more or less) standard way to convert NULL to int.

So it should be `(nEventsPerJob|0)` and `(nFilesPerJob|1)` like I supposed, right? If yes, then i have no further objections.

Reply:
Fixed now.